apps/src/templates/teacherNavigation/TeacherNavigationRouter.tsx

```diff
@@ -338,6 +340,12 @@ const TeacherNavigationRouter: React.FC<TeacherNavigationRouterProps> = ({
                 )
               }
             />
+            {DCDO.get('skills-dashboard', false) && (
```

Comment:
Curious why DCDO as opposed to experiment flag?

Reply:
Because experiment flags can be enable by a user without admin permissions, someone could read our codebase and add the url flag. But using DCDO requires a user to have admin permissions.

Not that there's anything really secret here, but better safe than sorry?

Reply:
ah - I see. I was more thinking that an experiment flag is specific to single user, but DCDO would turn the skills dashboard UI "on" for anyone viewing the site in that environment.